features/login.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Allow user to login into marketcube
+
+    AS a user,
+    I want to login into marketcube
+
+    # Background: Seller-user goes to marketcube login page
+
+
+    Scenario: Seller-User verify the components
+        Given User hits marketcube url
```

Comment:
this is a common statement for all scenarios so you have to write in 'Background'.